doc/api/documentation.md

```diff
@@ -39,15 +39,18 @@ The stability indices are as follows:
 
 ```txt
 Stability: 0 - Deprecated
-This feature is known to be problematic, and changes are
-planned.  Do not rely on it.  Use of the feature may cause warnings.  Backwards
-compatibility should not be expected.
+This feature is known to be problematic, and changes are planned. Do
+not rely on it. Use of the feature may cause warnings to be emitted.
+Backwards compatibility should not be expected.
 ```
 
 ```txt
 Stability: 1 - Experimental
-This feature is subject to change, and is gated by a command line flag.
-It may change or be removed in future versions.
+This feature is still under active development and subject to non-backwards
+compatible changes, or even removal, in any future version. Use of the feature
+is not recommended in production environments. The feature may only be usable
```

Comment:
nit:
s/The feature may only be usable The feature may only be usable/The feature may require the use of a command-line flag to enable it